conda_tutorial.md

```diff
@@ -0,0 +1,184 @@
++ Conda can quickly install, run, and update packages and their dependencies.
++ Conda can create, save, load, and switch between project specific software environments on your local computer.
+
+![](_static/conda1.png)
+
+Conda as a package manager helps you find and install packages. If you need a package that requires a different version of Python, you do not need to switch to a different environment manager, because Conda is also an environment manager. With just a few commands, you can set up a totally separate environment to run that different version of Python, while continuing to run your usual version of Python in your normal environment.
+
+
+# How does Conda work
+![](_static/conda2.png)
+
+# Benefits of Conda
+
+![](_static/conda3.png)
+
+
+
+## Installing & Activating Conda
+We have already installed conda on this instance, but please see the [installation HackMD](https://hackmd.io/im0eDxViQgmxTL4zYBfGoQ) after the class if you'd like to do this yourself.
+
+To be able to use that installation, we need to let the instance know what path to find bioconda in:
+
+```{bash}
+echo export PATH=$PATH:/opt/miniconda3/bin >> ~/.bashrc
+```
+
+Then, run the following command (or start a new terminal session) in order to activate the conda environment:
+
+```
+source ~/.bashrc
+```
+
+
+
+## Adding a new environment
+
+Check your current version of python by exectuting `python --version`
+
+To create a new environment named, for instance mynewenv (you can name it what ever you like), that includes, let’s say, a Python version 3.4., run:
+
+```
+conda create --name mynewenv python=3.4
+```
+
+## Activating and leaving (deactivating) an environment
+
+Inside a new Conda installation, the root environment is activated by default, so you can use it without activation.
+
+In other cases, if you want to use an environment (for instance manage packages, or run Python scripts inside it) you need to first activate it.
+
+```
+source activate mynewenv
+```
+The command prompt changes upon the environment’s activation, it now contains the active environment’s name.
+
+The directories of the active environment’s executable files are added to the system path (this means that you can now access them more easily). You can leave an environment with this command:
+
+```
+source deactivate
+```
+
+It needs to be mentioned that upon deactivating an environment, the root environment becomes active automatically.
+
+
+## What are Conda channels?
+
+Channels are the locations of the repositories where Conda looks for packages. Upon Conda’s installation, Continuum’s (Conda’s developer) channels are set by default, so without any further modification, these are the locations where your Conda will start searching for packages.
+
+Channels exist in a hierarchical order. The channel with the highest priority is the first one that Conda checks, looking for the package you asked for. You can change this order, and also add channels to it (and set their priority as well).
```

Comment:
i'm not sure its hierarchical...just ordered? I mean they are sort of hierarchical but nothing can be at the same level, so I think ordered might be more clear.